src/index.js

```diff
@@ -24,6 +24,7 @@ class Cropper extends React.Component {
   rafZoomTimeout = null
   state = {
     cropSize: null,
+    onWheelStart: false,
```

Comment:
I would name it something like `hasWheelJust Started` because current naming made me think it was a callback. 😀